my_entrypoint_prod.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+poetry run gunicorn --bind 0.0.0.0 "todo_app.app:create_app()"
```

Comment:
When I ran this file, I got a warning that said:
```
./my_entrypoint_prod.sh: line 2: $'\r': command not found
```

This is kind-of my problem, in that my Git setup on Windows is assuming it should add WIndows line endings to files even when it turns out it's wrong (because this will be copied into the Dockerfile and our container is Linux based) but it's something that we can avoid for future users by adding a `.gitattributes` file to the top level of the project to specify that `.sh` files should always retain Linux line endings 
For example, you can [look at this file that we use for a workshop to achieve the same thing](https://github.com/corndeladmin/DevOps-Course-Workshop-Module-10-Databases/blob/main/Globex-Database/.gitattributes):
```
*.csv eol=lf
wait-for-it.sh eol=lf
*.sql diff
*.sh eol=lf
```
"eol" means  end-of-line here, so you'd just need the final entry here to commit to lf endings